TestCases/compliance-level-3/0068-feel-equality/0068-feel-equality.dmn

```diff
@@ -0,0 +1,746 @@
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="context_003"/>
+        <literalExpression>
+            <text>{foo: "bar", bar: "baz"} = {bar: "baz", foo: "bar"}</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="context_004" id="_context_004">
+        <description>Tests FEEL expression: '{foo: "bar"} = {"foo": "bar"}' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression '{foo: "bar"} = {"foo": "bar"}'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="context_004"/>
+        <literalExpression>
+            <text>{foo: "bar"} = {"foo": "bar"}</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="context_005" id="_context_005">
+        <description>Tests FEEL expression: '{foo: "bar"} = {foo: "baz"}' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression '{foo: "bar"} = {foo: "baz"}'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="context_005"/>
+        <literalExpression>
+            <text>{foo: "bar"} = {foo: "baz"}</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="date_001" id="_date_001">
+        <description>Tests FEEL expression: 'date("2018-12-08") = date("2018-12-08")' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'date("2018-12-08") = date("2018-12-08")'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="date_001"/>
+        <literalExpression>
+            <text>date("2018-12-08") = date("2018-12-08")</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="date_002" id="_date_002">
+        <description>Tests FEEL expression: 'date("2018-12-07") = date("2018-12-08")' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'date("2018-12-07") = date("2018-12-08")'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="date_002"/>
+        <literalExpression>
+            <text>date("2018-12-07") = date("2018-12-08")</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="date_003" id="_date_003">
+        <description>Tests FEEL expression: 'date("2018-12-07") = date and time("2018-12-08")' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'date("2018-12-07") = date and time("2018-12-08")'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="date_003"/>
+        <literalExpression>
+            <text>date("2018-12-07") = date and time("2018-12-08")</text>
```

Comment:
This test case `date_003` but also `date_004`, `ym_duration_006`, and others which I may have missed are testing edgecase/errorcase/unspecified from the spec:
![image](https://user-images.githubusercontent.com/1699252/51735779-92ad9000-2088-11e9-93b3-e600a6364bed.png)

In my perspective these are beyond scope and better left out of testing.